content/applications/sales/point_of_sale/payment/six.rst

```diff
@@ -1,54 +1,55 @@
-====================================================
-Connect a SIX Payment Terminal to your PoS
-====================================================
+=========================================
+Connect a SIX Payment Terminal to the PoS
+=========================================
 
-Connecting a SIX payment terminal allows you to offer a fluid
-payment flow to your customers and ease the work of your cashiers.
+Connecting a SIX payment terminal allows you to offer a fluid payment flow to your customers and
+ease the work of your cashiers.
 
 .. warning::
-   Even though Worldline has recently acquired SIX Payment Services and both
-   companies use Yomani payment terminals, the firmware they run is different.
-   Terminals received from Worldline are therefore not compatible with this
-   integration.
+   Even though Worldline has recently acquired SIX Payment Services and both companies use Yomani
+   payment terminals, the firmware they run is different. Terminals received from Worldline are
+   therefore not compatible with this integration.
 
 Configuration
 =============
 
 Configure the Payment Method
 ----------------------------
 
-First, make sure that the POS Six module is installed. For this, go to *Apps*,
-remove the "Apps" filter and search for "POS Six".
+First, make sure that the :guilabel:`POS Six`` module is installed. For this, go to
+:guilabel:`Apps`, remove the *Apps* filter, and search for *POS Six*.
 
-.. image:: media/six_01.png
-    :align: center
+.. image:: six/pos-six-module.png
+   :align: center
+   :alt: The POS Six module.
 
-Back in :menuselection:`Point of Sale --> Configuration --> Payment Methods`,
-Create a new payment method for SIX, select the payment terminal option
-"SIX without IoT Box", and enter your payment terminal IP address.
+Back in :menuselection:`Point of Sale --> Configuration --> Payment Methods`, create a new payment
+method for SIX, select the payment terminal option :guilabel:`SIX`, and enter the payment terminal
+IP address.
 
-.. image:: media/six_02.png
-    :align: center
+.. image:: six/new-payment-method-six.png
```

Comment:
The file size is fine but you'll probably get dinged for 4k resolution during technical review.
Would reduce file width by about 50% if you can to around tablet breakpoint of ~787px (as opposed to 1500+ pixels"